src/sentry/runner/commands/presenters/audit_log_presenter.py

```diff
@@ -0,0 +1,45 @@
+import itertools
+import logging
+from datetime import datetime, timezone
+
+from sentry import options
+from sentry.flags.providers import FlagAuditLogItem, handle_flag_pole_event_internal
+from sentry.runner.commands.presenters.webhookpresenter import WebhookPresenter
+
+logger = logging.getLogger()
+
+
+class AuditLogPresenter(WebhookPresenter):
+    @staticmethod
+    def is_webhook_enabled() -> bool:
+        return (
+            options.get("flags:options-audit-log-is-enabled") is True
+            and options.get("flags:options-audit-log-organization-id") is not None
+        )
+
+    def flush(self) -> None:
+        if not self.is_webhook_enabled():
+            logger.warning("Options audit log webhook is disabled.")
+            return None
+
+        items = self._create_audit_log_items()
+        handle_flag_pole_event_internal(
+            items, organization_id=options.get("flags:options-audit-log-organization-id")
+        )
+
+    def _create_audit_log_items(self) -> list[FlagAuditLogItem]:
+        return [
+            {
+                "action": action,
+                "created_at": datetime.now(tz=timezone.utc),
+                "created_by": "internal",
+                "flag": flag,
+                "tags": tags,
+            }
+            for flag, action, tags in itertools.chain(
+                ((flag, "created", {"value": v}) for flag, v in self.set_options),
+                ((flag, "deleted", {}) for flag in self.unset_options),
+                ((flag, "updated", {"value": v}) for flag, _, v in self.updated_options),
+                ((flag, "updated", {}) for flag, _ in self.drifted_options),
```

Comment:
what does a drifted option mean?

Reply:
Internal lingo.  Basically the value was updated but the existing value in the db was unexpected.

Reply:
we don't care about the new updated value in this case?

Reply:
We still do.  The value is changed in the db.  Its just that the old value was not what we expected (someone manually changed it).

Reply:
Oh I see the tags.  The drifted options don't contain the new value which is why it was omitted.  This is a legacy event any way.  I could probably remove it.